libs/api/domains/education/src/lib/mock-data/my-family.ts

```diff
@@ -0,0 +1,77 @@
+import { ISLFjolskyldan } from '@island.is/clients/national-registry-v1'
+
+// Note: this family is generated from a real family, but the names and
+// national IDs have been changed to protect the identity of said family.
+// Extra note: it would be nice to have more family cases to test.
+export const MyFamilyMock = [
+  {
+    attributes: {
+      'diffgr:id': 'ISLFjolskyldan2',
+      'msdata:rowOrder': '1',
+      'diffgr:hasChanges': 'inserted',
+    },
+    Kennitala: '0910819979',
```

Comment:
Let's make sure all of these kt's are invalid since we don't want real people's kts to be here (even by accident) if we can avoid it. There's a lint warning for this btw.

Reply:
yup, i know. Though those are not real national IDs, even though they are valid.

The problem is that the service uses the kennitala package to determine people's age, and the age doesn't work for invalid kennitala.

But I'll mock the package and ping you when it's done. Then I can get rid of this stuff.

Reply:
yeah, it was a bit of a headache to mock the package, mostly with having it play nicely with typescript, but the result is not __too__ bad 😮 maybe.. anyways.. i'd appreciate a second glance.

agree re: whitelisted national IDs, possibly for other stuff as well.. but generating the non-real-but-valid national IDs was messy too, and being able to have the mock handle all cases in our tests as well is kind of nice.